Leanplum/Utils/LPExceptionHandler.h

```diff
@@ -0,0 +1,35 @@
+//
+//  LPExceptionHandler.h
```

Comment:
minor thing, the headers look really different when copied over from old sdk, do we care about this?

Reply:
Nope, we dont care. Unless the copied code is with a external apache license.